spec/dummy/client/app/startup/ServerReduxApp.jsx

```diff
@@ -15,7 +15,7 @@ import reducers from '../reducers/reducersIndex';
 
 import HelloWorldContainer from '../components/HelloWorldContainer';
 
-let App = props => {
+let ReduxApp = props => {
```

Comment:
Also should be `const`. And to save some typing and show what's exactly we're exporting here, we can skip variable declaration and just export the function:

```js
export default props => {
  // ...
}
```